src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -432,6 +462,156 @@ public async Task CreatePasswordPolicyRule()
+                Type = PolicyType.AccessPolicy.ToString(),
+                Actions = new AccessPolicyRuleActions
+                {
+                      AppSignOn = new AccessPolicyRuleApplicationSignOn
+                      {
+                          Access = "DENY",
+                          VerificationMethod = new VerificationMethod
+                          {
+                              Type = "ASSURANCE",
+                              FactorMode = "1FA",
+                              ReauthenticateIn = "PT43800H",
+                          },
+                      },
+                },
+            };
+
+            var createdPolicyRule =
+                await client.Policies.AddPolicyRuleAsync<AccessPolicyRule>(accessPolicyRuleOptions, accessPolicyId);
+            try
+            {
+                createdPolicyRule.Should().NotBeNull();
+                createdPolicyRule.Name.Should().Be(accessPolicyRuleOptions.Name);
+                createdPolicyRule.Actions.Should().NotBeNull();
+                createdPolicyRule.Actions.AppSignOn.Access.Should().Be("DENY");
+                createdPolicyRule.Actions.AppSignOn.VerificationMethod.Type.Should().Be("ASSURANCE");
+                createdPolicyRule.Actions.AppSignOn.VerificationMethod.FactorMode.Should().Be("1FA");
+                createdPolicyRule.Actions.AppSignOn.VerificationMethod.ReauthenticateIn.Should().Be("PT43800H");
+                createdPolicyRule.Type.Should().Be(PolicyType.AccessPolicy);
+            }
+            finally
+            {
+                await client.Applications.DeactivateApplicationAsync(createdApp.Id);
+                await client.Applications.DeleteApplicationAsync(createdApp.Id);
+            }
+        }
+
+        [Fact]
+        public async Task CreateProfileEnrollmentPolicyRule()
+        {
+            var client = TestClient.Create();
+
+            var policy = new Policy()
+            {
+                // Name has a maximum of 50 chars
+                Name = $"dotnet-sdk: ProfileEnrollmentPolicy {Guid.NewGuid()}".Substring(0, 50),
+                Type = PolicyType.ProfileEnrollment,
+                Status = "ACTIVE",
+                Description = "The default policy applies in all situations if no other policy applies.",
+            };
+
+            var createdPolicy = await client.Policies.CreatePolicyAsync(policy);
+            var defaultPolicyRule = await createdPolicy.ListPolicyRules().OfType<IProfileEnrollmentPolicyRule>().FirstOrDefaultAsync();
+            var profileAttributes = new List<IProfileEnrollmentPolicyRuleProfileAttribute>();
+            profileAttributes.Add(new ProfileEnrollmentPolicyRuleProfileAttribute
+            {
+                Name = "email",
+                Label = "Email",
+                Required = true,
+            });
+
+            defaultPolicyRule.Actions.ProfileEnrollment.Access = "ALLOW";
```

Comment:
Actions, profile enrollment are being set here and the code segment that follows below (lines 575 to 589) is setting it again (twice).

Reply:
Good catch. Fixed.